Makefile

```diff
@@ -34,18 +35,20 @@ SHELL:=/bin/bash
 	if test -f nxcomp/Makefile; then ${MAKE} -C nxcomp $@; fi
 	if test -f nxproxy/Makefile; then ${MAKE} -C nxproxy $@; fi
 	if test -d nx-X11; then \
+	    if test -f nx-X11/lib/Makefile; then ${MAKE} -C nx-X11/lib $@; fi; \
 	    if test -f nxcompshad/Makefile; then ${MAKE} -C nxcompshad $@; fi; \
 	    if test -f nx-X11/Makefile; then ${MAKE} -C nx-X11 $@; fi; \
 	fi
 
 	# clean auto-generated files
 	if [ "x$@" == "xclean" ] || [ "x$@" = "xdistclean" ]; then \
-	    ./mesa-quilt pop -a; \
+	    if [ -x ./mesa-quilt ]; then ./mesa-quilt pop -a; fi; \
```

Comment:
Nope. if then clauses don't fail, where as ``[...] && do_somthing`` creates a non-zero exit code if the condition is not met.

Not changing this.